src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -100,6 +101,17 @@ sealed trait Task {
 
 object Task {
 
+  object Implicits {
+    import scala.language.implicitConversions
+
+    implicit def tasksToAllPlaced(tasks: Seq[Task]): Seq[Placed] = tasks.map(taskAsPlaced(_))(collection.breakOut)
```

Comment:
We try to avoid implicit type conversions, since this happens "magically" behind the scenes.
```
val task: Task
def foo(placed: Placed) {}
foo(task)
```

What about defining Task as Placed?